okhttp/build.gradle.kts

```diff
@@ -213,48 +214,62 @@ android {
rnal;okhttpinternal=true;mandatory:=okhttpinternal;uses:="javax.net.ssl,kotlin,kotlin.annotation,kotlin.jvm.functions,kotlin.reflect,okhttp3,okhttp3.internal.concurrent,okhttp3.internal.connection,okhttp3.internal.http2,okio";version="5.0.0",okhttp3.internal.authenticator;okhttpinternal=true;mandatory:=okhttpinternal;uses:="kotlin,kotlin.jvm.internal,okhttp3";version="5.0.0",okhttp3.internal.cache;okhttpinternal=true;mandatory:=okhttpinternal;uses:="kotlin,kotlin.jvm.functions,kotlin.jvm.internal,kotlin.jvm.internal.markers,kotlin.text,okhttp3,okhttp3.internal.concurrent,okio";version="5.0.0",okhttp3.internal.cache2;okhttpinternal=true;mandatory:=okhttpinternal;uses:="kotlin,kotlin.jvm.internal,okio";version="5.0.0",okhttp3.internal.concurrent;okhttpinternal=true;mandatory:=okhttpinternal;uses:="kotlin,kotlin.jvm.functions,kotlin.jvm.internal";version="5.0.0",okhttp3.internal.connection;okhttpinternal=true;mandatory:=okhttpinternal;uses:="javax.net.ssl,kotlin,kotlin.collections,kotlin.jvm.functions,kotlin.jvm.internal,okhttp3,okhttp3.internal.concurrent,okhttp3.internal.http,okhttp3.internal.http2,okhttp3.internal.ws,okio";version="5.0.0",okhttp3.internal.graal;okhttpinternal=true;mandatory:=okhttpinternal;uses:="com.oracle.svm.core.annotate,kotlin,okhttp3.internal.platform,org.graalvm.nativeimage.hosted";version="5.0.0",okhttp3.internal.http;okhttpinternal=true;mandatory:=okhttpinternal;uses:="kotlin,kotlin.jvm,kotlin.jvm.internal,okhttp3,okhttp3.internal.connection,okio";version="5.0.0",okhttp3.internal.http1;okhttpinternal=true;mandatory:=okhttpinternal;uses:="kotlin,kotlin.jvm.internal,okhttp3,okhttp3.internal.http,okio";version="5.0.0",okhttp3.internal.http2;okhttpinternal=true;mandatory:=okhttpinternal;uses:="kotlin,kotlin.enums,kotlin.jvm.functions,kotlin.jvm.internal,okhttp3,okhttp3.internal.concurrent,okhttp3.internal.http,okhttp3.internal.http2.flowcontrol,okio";version="5.0.0",okhttp3.internal.http2.flowcontrol;okhttpinternal=true;mandatory:=okhttpinternal;uses:=kotlin;version="5.0.0",okhttp3.internal.idn;okhttpinternal=true;mandatory:=okhttpinternal;uses:="kotlin,kotlin.jvm.functions,okio";version="5.0.0",okhttp3.internal.platform;okhttpinternal=true;mandatory:=okhttpinternal;uses:="javax.net.ssl,kotlin,kotlin.jvm,kotlin.jvm.internal,okhttp3,okhttp3.internal.tls,org.conscrypt";version="5.0.0",okhttp3.internal.proxy;okhttpinternal=true;mandatory:=okhttpinternal;uses:=kotlin;version="5.0.0",okhttp3.internal.publicsuffix;okhttpinternal=true;mandatory:=okhttpinternal;uses:="kotlin,kotlin.jvm.internal,okio";version="5.0.0",okhttp3.internal.tls;okhttpinternal=true;mandatory:=okhttpinternal;uses:="javax.net.ssl,kotlin,kotlin.jvm.internal";version="5.0.0",okhttp3.internal.url;okhttpinternal=true;mandatory:=okhttpinternal;uses:="kotlin,okio";version="5.0.0",okhttp3.internal.ws;okhttpinternal=true;mandatory:=okhttpinternal;uses:="kotlin,kotlin.jvm.internal,okhttp3,okhttp3.internal.concurrent,okhttp3.internal.connection,okio";version="5.0.0"""",
```

Comment:
WHAT IN THE ABSOLUTE HORROR IS THIS?!

Reply:
The Kotlin update broke the build, but showed that some tactical workarounds were not effective. I don't have a better current solution than this.

We do have tests covering the result of this build, otherwise I would be very nervous.

There is no current plan to support KMP https://github.com/bndtools/bnd/issues/6417

And I hit this issue using the plugin https://github.com/bndtools/bnd/issues/6590

Reply:
https://github.com/square/okhttp/pull/8810